target/scripts/helpers/setup-rspamd.sh

```diff
@@ -36,8 +36,15 @@ function __rspamd__preflight_checks
 # `smtpd_milters` in `main.cf`.
 function __rspamd__adjust_postfix_configuration
 {
-  # shellcheck disable=SC2016
-  sed -i -E 's|^(smtpd_milters =.*)|\1 inet:localhost:11332|g' /etc/postfix/main.cf
+  postconf 'rspamd_milter = inet:localhost:11332'
+
+  if grep -q -E '^smtpd_milters.*=.*rspamd_milter' /etc/postfix/main.cf
+  then
+    _log 'warn' "'smtpd_milters' already contains Rspamd milter, it will not be added twice - likely an inconsistency (did you run docker compose down properly?)"
+  else
+    # shellcheck disable=SC2016
+    sed -i -E 's|^(smtpd_milters =.*)|\1 \$rspamd_milter|g' /etc/postfix/main.cf
+  fi
```

Comment:
A warning like that doesn't seem necessary to communicate?

<details>
<summary>Ignore, postconf is a smarter approach</summary>

The DB helper has a method that could be re-used for checking duplicates I think?:

https://github.com/docker-mailserver/docker-mailserver/blob/199e3c77214290f16b97dcccb97c1c3a6a5d0aeb/target/scripts/helpers/database/db.sh#L130-L140

</details>

but we could also do this:

```bash
# Method to update settings while avoiding adding duplicates
function _maincf_append_value_to_key_if_missing()
{
  local KEY=$1
  local VALUE_TO_APPEND=$2

  # Extract value of key from main.cf:
  local VALUE=$(sed -nE "s/^${KEY} = (.*)/\1/p" /etc/postfix/main.cf)

  if ! grep -q -F "${VALUE_TO_APPEND}" <<< "${VALUE}"
  then
    postconf "${KEY} = ${VALUE} ${VALUE_TO_APPEND}"
  fi
}

# ....

_maincf_append_value_to_key_if_missing 'smtpd_milters' '$rspamd_milter'
_maincf_append_value_to_key_if_missing 'smtpd_milters' '$dmarc_milter'
_maincf_append_value_to_key_if_missing 'smtpd_milters' '$dkim_milter'
_maincf_append_value_to_key_if_missing 'non_smtpd_milters' '$dkim_milter'
```

Names of things could perhaps be better, but simple enough to re-use:
- Give it a key and a value to append
- If there's a match for that key, then append the value via `postconf` :sunglasses: (_it'll ensure consistent white-space formatting between key and values_)